contrib/soju/files/soju

```diff
@@ -0,0 +1,8 @@
+# soju system service
+
+type       = process
+command    = /usr/bin/capsh --inh=cap_net_bind_service --user=_soju -- -c "exec /usr/bin/soju -config /etc/soju/config"
```

Comment:
`Unable to set group list for user: Operation not permitted`

Reply:
i guess that would be a reason :)  

the existing thing you wrote (`--inh="cap_net_bind_service+eip"`) is a syntax error

Reply:
actually no i just typed it wrong

Reply:
but anyway, this doesn't work. as the documentation says `--inh` doesn't actually grant a capability. if you put `listen http+prometheus://localhost:412` it will fail (this cap is only needed for low (<1024) port numbers)

Reply:
did a bit of reading and it seems you need something like  

```
capsh --caps="cap_net_bind_service,cap_setgid,cap_setuid,cap_setpcap+eip" --user=_soju --addamb='cap_net_bind_service'
```

this then:  

```
$ doas grep Cap /proc/432/status
CapInh:	00000000000005c0
CapPrm:	0000000000000400
CapEff:	0000000000000400
CapBnd:	000001ffffffffff
CapAmb:	0000000000000400
$ capsh --decode=0000000000000400
0x0000000000000400=cap_net_bind_service
```

only gives it that one. found this out via https://www.redhat.com/en/blog/linux-capabilities-in-openshift

very unintuitive :P

Reply:
ah, and the setpcap one isn't needed

Reply:
makes sense